commonjs/.eslintrc.json

```diff
@@ -1,8 +1,5 @@
 {
-  "extends": [
-    "../base/.eslintrc.json",
-    "plugin:prettier/recommended"
-  ],
+  "extends": ["../base/.eslintrc.json"],
```

Comment:
All the configs need to extend prettier otherwise they overwrite the prettier rules.